lib/assert.js

```diff
@@ -47,16 +40,11 @@ const assert = module.exports = ok;
 
 // All of the following functions must throw an AssertionError
 // when a corresponding condition is not met, with a message that
-// may be undefined if not provided.  All assertion methods provide
+// may be undefined if not provided. All assertion methods provide
 // both the actual and expected values to the assertion error for
 // display purposes.
 
-function fail(actual, expected, message, operator, stackStartFunction) {
-  if (arguments.length === 1)
-    message = actual;
-  if (arguments.length === 2)
-    operator = '!=';
-  const errors = lazyErrors();
+function _fail(actual, expected, message, operator, stackStartFunction) {
```

Comment:
I'm not a huge fan of renaming all of these functions but I don't have strong feelings about it, so now it's `inner*`.